packages/typespec-azure-resource-manager/src/operations.ts

```diff
@@ -205,7 +211,8 @@ export function $armRenameListByOperation(
     [parentTypeName, parentFriendlyTypeName] = getArmParentName(context.program, resourceType);
   }
   const parentType = getParentResource(program, resourceType);
-  if (parentType) {
+
+  if (parentType && !isArmBuiltInResource(program, parentType)) {
```

Comment:
We originally talked about allowing non-resource model in @parentResource to participate in path resolution for this fix. Are we to track and fix that separately now?

Reply:
That is actually what this does.  If you apply the `@armBuiltInResource` decorator to a non-resource type with a path key and segment property, it can be made the parent of any resource.  It is basically how you mark a virtual resource that nas no operations in the path